src/layouts/Skeleton.astro

```diff
@@ -9,6 +9,7 @@ interface Props {
 
 const { title, description } = Astro.props;
 const IS_PRODUCTION = import.meta.env.PROD;
+const pageTitle = `${title} | VeraId`;
```

Comment:
I think it's a bit of an overkill to create a variable for this, when we won't be using it anywhere else.